concourse/scripts/test_and_build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash -l
```

Comment:
Shouldn't `concourse/scripts/test.sh` be deleted?

Reply:
yes